fellow-submissions/stephanie-smith/style.css

```diff
@@ -0,0 +1,90 @@
+body {
+  background-color: #13293D;
+  font-family: Inter, Roboto, sans-serif;
+  margin: 0;
+  padding: 0 100px;
+  color: #E8F1F2;
+}
+
+h1 {
+  margin: 0;
+}
+
+.heading {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  background-color: #006494;
+}
+
+.heading h1 {
+  margin: 0 0 -5px 0;
+  padding-top: 20px;
+  font-size: 70px;
+  font-weight: normal;
+}
+
+.heading p {
+  font-size: 30px;
+}
+
+.container {
+  display: grid;
+  grid-template-columns: 300px 74.3%;
```

Comment:
Where did these two column width values come from? The reason why your `.container` is extending past the length of its container is because `300px` + `74.3%` doesn't always add to 100%. Think about it like this, if your grid container (`.container`) is  100px based on the window size, column 1 will be 300px and column 2 will be 74.3px. Total will be 374.3px and it will extend past the container. See pic attached for reference. 
<img width="446" alt="Screen Shot 2019-10-06 at 4 44 01 PM" src="https://user-images.githubusercontent.com/11546505/66275628-91a84180-e858-11e9-9e87-8c33c2cd05e1.png">
Try using fractional units instead. 
```css
grid-template-columns: 1fr 3fr;
```
might be a good starting point.